config.rb

```diff
@@ -2,6 +2,9 @@
 
 GovukTechDocs.configure(self)
 
+# use custom layout file
+set :layout, 'custom'
```

Comment:
Do you know where `custom` as an option comes from? I couldn't see it as an option here: https://github.com/search?q=repo%3Aalphagov%2Ftdt-documentation%20layout&type=code (I mean it seems to work, I just can't work out what it does)

Reply:
This is setting the default layout to the [`custom.erb` layout file](https://github.com/alphagov/wcag-primer/blob/884df1d35102ac76b369125f851516fbd1b4858a/source/layouts/custom.erb) that the PR adds.